lib/linguist/heuristics.rb

```diff
@@ -226,6 +226,16 @@ def call(data)
       end
     end
 
+    disambiguate ".gml" do |data|
+      if /^\s*(\<\?xml|xmlns)/i.match(data)
+        Language["XML"]
+      elsif /^\s*(graph|node)\s+\[$/i.match(data)
```

Comment:
We still want to match on:
```
graph
[
    foobar
```
right?
Any reason you changed the `[\s\n\r]`?

Reply:
I thought \s would include \n and \r, hence I removed it

Reply:
``\s matches any whitespace character (equal to [\r\n\t\f\v ])``

Reply:
This format is in sample.gml and now also in sample3.gml. I think the succeeding tests should prove that the regex works, right? :)

Reply:
Oh, right! my bad!

Reply:
My bad for initially writing \s\r\n ;)